interfaces/Portalicious/src/app/app.routes.ts

```diff
@@ -42,6 +42,7 @@ export const routes: Routes = [
   },
   {
     path: AppRoutes.userSettings,
+    title: $localize`:Browser-tab-title@@page-title-user-settings:Change password`,
```

Comment:
I think this might be the only page with a title, and therefore also why it's the only page that persists the title (ie. the bug we saw this morning in the call).